### ISSUE TITLE
Automatically adjust SunSky atmosphere radius by camera height.

### ISSUE THREAD

**Issue by nithinp7** [CesiumGS/cesium-unreal#282]
We should add an option to switch to a larger atmosphere radius as the camera gets to higher altitudes. This will help prevent some of the strange diagonal artifacts we see currently when zooming out to see the whole planet. The artifacts are due to the SunSky being spherical while the actual planet is ellipsoidal. iirc, project anywhere may have implemented a technique like this somewhere, it may be worth looking for that implementation.

**Comment by argallegos** [comment]
I agree, but wanted to add a note - we need to be careful with the values we adjust this to. It's inevitable that increasing the atmosphere radius will result in the atmosphere not aligning with the terrain properly, and its not noticeable at low increases but becomes very obvious very quickly. If we were able to change the atmosphere origin to be more centered over the entire world, it would be less of a problem because it would at least look uniform. 

Additionally, I've noticed that a value that looks good in, say, scene view doesn't necessarily look good in a rendered sequence or other viewing method. I had to increase the atmosphere size significantly more than I thought I did based on the scene view in order for the issue to go away in a render.
